migrate/migrate.go

```diff
@@ -17,6 +18,8 @@ import (
 	pipep "github.com/heetch/migrate/pipe"
 )
 
+var nowFunc = time.Now
```

Comment:
I'm not sure I understand your point?

For me the `nowFunc` is only here to be able stub the time, like `timecop` in rails http://labs.yulrizka.com/en/2014/10/stubbing-time-dot-now-in-golang.html#.VcsJxxOqpBc